packages/calling-component-bindings/src/handlers/createCommonHandlers.ts

```diff
@@ -817,6 +897,14 @@ export const createDefaultCommonCallingHandlers = memoizeOne(
       onMuteAllRemoteParticipants,
       onAcceptCall: notImplemented,
       onRejectCall: notImplemented,
+      /* @conditional-compile-remove(together-mode) */
+      onCreateTogetherModeStreamView,
+      /* @conditional-compile-remove(together-mode) */
+      onStartTogetherMode: notImplemented,
+      /* @conditional-compile-remove(together-mode) */
+      onDisposeTogetherModeStreamViews,
+      /* @conditional-compile-remove(together-mode) */
+      onSetTogetherModeSceneSize,
```

Comment:
should this also be included in the type above?

Reply:
onSetTogetherModeSceneSize is a callback just like onStartTogetherMode. So far, we don't need it as a type